src/+datools/+observation/+operator/Indexed.m

```diff
@@ -1,7 +1,8 @@
-classdef Indexed < datools.observation.Observation
+classdef Indexed < datools.observation.operator.Observation
```

Comment:
Suggestion: Have subclass name include superclass name, e.g. `IndexedObservation`

Reply:
This class might be more useful as a "decorator" to add indexing capability to any other observation class.  This would remove the redundancy of indexing in the nonlinear observation class.

Reply:
https://en.wikipedia.org/wiki/Decorator_pattern

Reply:
> Suggestion: Have subclass name include superclass name, e.g. `IndexedObservation`

I agree with this 100%

Reply:
> This class might be more useful as a "decorator" to add indexing capability to any other observation class. This would remove the redundancy of indexing in the nonlinear observation class.

If we can do that without a performance hit, then I am all for it.